src/MdocDataTransferEvent.ts

```diff
@@ -5,7 +5,12 @@ export enum MdocDataTransferEvent {
 
 export type OnResponseSendPayload = null
 
-export type OnRequestReceivedEventPayload<T = Array<number>> = {
+export type OnRequestReceivedEventPayload<T = string> = {
   deviceRequest: T
```

Comment:
might be nice to also call these based on the encoding, so deviceRequestBase64 or deviceRequestHex